api/model/article.js

```diff
@@ -15,6 +15,17 @@ exports.getAll = function (request, response) {
   });
 };
 
+exports.deleteArticle = function (request, response) {
```

Comment:
Don't you think if we call it as `delete` only a good thing? I say this because it is on the `article.js` file, dealing with the Article model.